Runtime/CoreLib.Script/Array.js

```diff
@@ -207,3 +207,23 @@ ss.arrayFill = function#? DEBUG ss$arrayFill##(dst, val, index, count) {
 			dst[index + count] = val;
 	}
 };
+
+ss.arrayCopy = function#? DEBUG ss$arrayCopy##(src, spos, dst, dpos, len) {
+	if (!ss.isValue(src) || !ss.isValue(dst))
+		throw new ss_NullReferenceException();
+
```

Comment:
This should be an ArgumentNullException for .net compatibility, but IMO we can skip argument validation since we don't do that for most methods in order to keep the runtime lighter.